DXRFixes/DeusEx/Classes/AutoTurretGun.uc

```diff
@@ -0,0 +1,18 @@
+class AutoTurretGun injects AutoTurretGun;
+
+auto state Active
+{
+    function TakeDamage(int Damage, Pawn EventInstigator, vector HitLocation, vector Momentum, name DamageType)
+    {
+        local AutoTurret turret;
+
+        if (DamageType == 'NanoVirus') {
+            //Pass the "damage" along to the base.
+            //This makes it a bit easier to hit some tucked away turrets
+            turret = AutoTurret(Owner);
+            turret.HandleScrambler(EventInstigator,Damage);
```

Comment:
note this means the damage is typically applied twice, but that just means the crazed timer is longer than you might think?